# 
# @brief    : VISION_POSITION
# @time     : 2020/10/05
# @auhtor   : liuxu <liuxu.ccc@gmail.com> 
#

# position gotten from vision

geometry_msgs/Point board
geometry_msgs/Point head
float32 car_yaw